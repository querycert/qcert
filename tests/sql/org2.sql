select name
from employees
where age > 32
;
