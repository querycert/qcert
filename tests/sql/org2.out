[{"name": "Jim Does"}, {"name": "Joan Doe"}, {"name": "James Do"}]